commit e787d72ada78a1dac1654c4d3d530f50334e7846
Author: Sebastian Gottfried <sebastiangottfried@web.de>
Date:   Mon Mar 10 11:51:31 2014 +0100

    data: remove an untypable charaber from the Slovanian course
    
    BUG: 331945
    FIXED-IN: 4.12.4

diff --git a/data/courses/si.xml b/data/courses/si.xml
index fc5f3c2..07feeb1 100644
--- a/data/courses/si.xml
+++ b/data/courses/si.xml
@@ -201,7 +201,7 @@ odejo ne&#x10D;kinjin na&#x10D;&#x10D;ajo oljili oskalek
 naslednika na&#x10D;njajo&#x10D; noska oson&#x10D;a neodlo&#x10D;i&#x10D;ne
 nalijejo neolikank nanos oslinjeni ni&#x10D;li
 ojekleni nenadni ondi nadela neonkine okajen
-ni&#x10D;a nasekala naslada one&#x10D;dena neonka neidili&#xFFFD;n
+ni&#x10D;a nasekala naslada one&#x10D;dena neonka neidilin
 nasi&#x10D;nje na&#x10D;ne odsko&#x10D;le na&#x10D;kana osedlalo
 naslonijo oken&#x10D;a naslonilo okisalo odnosni
 naseljenkina odlo&#x10D;la nasilnik nedojena nenaklonjeni
